### PR TITLE
Simply map <c-w> to normal mode <c-w>

### PR DESCRIPTION
**Pull request by mcchrish**
I think we can just map <c-w> to normal mode <c-w> that wall all <c-w> will just work not just the four mappings we created.

Also I think it make sense that it should always be mapped regardless of `set_default_mappings`. `set_default_mappings` is if they want to get rid of `<leader>n`.

### CODE REVIEW

**Comment by N-R-K** [comment]
The change looks good. Although I feel like it should be guarded by some config var in case someone wants to use some other binding to <C-w>.

But if we want to remove the guard and enable it all the time, then we'll have to update the docs on that as well.

File: doc/nnn.txt [hunk 1]
```diff
@@ -96,9 +96,10 @@ Defaults mappings. Can be disabled by setting |g:nnn#set_default_mappings| to
```
Comment (N-R-K) [inline doc/nnn.txt:96]: Here it says that the binding can be disabled via `g:nnn#set_default_mappings` but that's not true with how `<C-w> is implemented currently.

**Comment by mcchrish** [comment]
I'm wondering if it also makes sense to map `:` to normal mode `:`. It's common enough to be added as default. What do you think?
Also are there more default mappings that can be handy?

**Comment by N-R-K** [comment]
> I'm wondering if it also makes sense to map : to normal mode :. It's common enough to be added as default. What do you think?

I was actually thinking about this as well. `nnn` doesn't seem to have any default binding set to `:` so it should be okay.

**Comment by mcchrish** [comment]
I realized it might not be ideal to map `:`. It will be triggered even during times like renaming/creating files or any other state nnn is in.

**Review by N-R-K — APPROVED** [review]
Looks good. Also like that `<C-w>` is configureable. Despite using very few plugins I've still encountered issues where a plugin was colluding with my keybinding so I'm always in favor of making things configureable.

**Pull request merged**